src/Uno.UI.Composition/Composition/CompositionSurfaceBrush.skia.cs

```diff
@@ -100,6 +100,7 @@ internal override void UpdatePaint(SKPaint fillPaint, SKRect bounds)
 				}
 
 				fillPaint.IsAntialias = true;
+				fillPaint.FilterQuality = SKFilterQuality.High;
```

Comment:
@Youssef1313 don't think we have a proper way to test this right? outside of doing a manual test

Reply:
It's probably *possible* but may be a bit hacky. Can look into it in a follow-up PR if necessary

Reply:
Yeah I think it would be great to see if we can prevent this long term with a proper test

Reply:
I spent a bit of time now but what I had in mind for testing didn't work out. :/

Reply:
Figured out a test, finally.